model/riscv_types.sail

```diff
@@ -390,6 +390,18 @@ enum amoop = {AMOSWAP, AMOADD, AMOXOR, AMOAND, AMOOR,
               AMOMIN, AMOMAX, AMOMINU, AMOMAXU}   /* AMO ops */
 enum csrop = {CSRRW, CSRRS, CSRRC}                /* CSR ops */
 
+enum brop = {RISCV_SH1ADD, RISCV_SH2ADD, RISCV_SH3ADD, RISCV_BCLR,
```

Comment:
As discussed, split these up so there are per-Zb* versions of each enum to avoid incomplete pattern match warnings

Reply:
Done.